hls4ml/converters/onnx/reshape.py

```diff
@@ -36,3 +38,29 @@ def parse_flatten_layer(node, input_names, input_shapes, graph):
     layer['target_shape'] = [-1]  # does not contain batch dimension
 
     return layer
+
+
+@onnx_handler('Resize')
+def parse_resize_layer(node, input_names, input_shapes, graph):
+    def get_scales(graph, name, scale_shapes):
+        for i in graph.initializer:
```

Comment:
I would prefer if this didn't access the initializer but instead accessed the last input node as an actual node. (It may need to be moved to the init function of the Resize).